packages/notifi-react-card/lib/hooks/SubscriptionCardConfig.ts

```diff
@@ -179,4 +179,22 @@ export type CardConfigItemV1 = Readonly<{
   eventTypes: EventTypeConfig;
   inputs: InputsConfig;
   contactInfo: ContactInfoConfig;
+  titles: TitleSubtitleConfig;
```

Comment:
for old configs, we'd be missing this configuration right? let's make it optional

alternatively, add some code to the deserializer in useSubscriptionCard so that it does more than just JSON.parse

Reply:
@juni-notifi addressed